.github/workflows/ruby.yml

```
@@ -34,6 +34,14 @@ jobs:
       env:
         UPLOADCARE_PUBLIC_KEY: demopublickey
         UPLOADCARE_SECRET_KEY: demoprivatekey
+    - name: Upload coverage reports to Codecov
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)